tvb_scripts/head/model/head.py

```diff
@@ -47,17 +49,21 @@ def cortex(self):
         return cortex
 
     def configure(self):
-        self.connectivity.configure()
-        self.cortical_surface.configure()
-        self.subcortical_surface.configure()
+        if isinstance(self.connectivity, Connectivity):
+            self.connectivity.configure()
+        if isinstance(self.cortical_surface, Surface):
+            self.cortical_surface.configure()
+        if isinstance(self.cortical_surface, Surface):
+            self.subcortical_surface.configure()
```

Comment:
These checks seem hackins. Do we really need them ?
I would try to leave without these